cmd/goal/clerk.go

```diff
@@ -155,6 +159,8 @@ func init() {
 	simulateCmd.Flags().StringVarP(&outFilename, "result-out", "o", "", "Filename for writing simulation result")
 	simulateCmd.Flags().BoolVar(&simulateAllowEmptySignatures, "allow-empty-signatures", false, "Allow transactions without signatures to be simulated as if they had correct signatures")
 	simulateCmd.Flags().BoolVar(&simulateAllowMoreLogging, "allow-more-logging", false, "Lift the limits on log opcode during simulation")
+	simulateCmd.Flags().BoolVar(&simulateAllowExtraOpcodeBudget, "allow-extra-opcode-budget", false, "Apply max extra budget for apps during simulation")
+	simulateCmd.Flags().Uint64Var(&simulateExtraOpcodeBudget, "extra-opcode-budget", 0, "Apply extra budget during simulation")
```

Comment:
nit: descriptions contain just "budget" instead of "opcode budget"

Reply:
see [more test in e2e for --allow-extra-opcode-budget, pr comment response](https://github.com/algorand/go-algorand/pull/5354/commits/2eebb810ca2052d22315fee59faa5f8aadd89940)

Reply:
It looks like in that commit you removed the word "opcode" from `simulateAllowExtraOpcodeBudget` and `simulateExtraOpcodeBudget`, which wasn't what I expected.

I thought based on prior conversation, it was decided we should always prefer saying "opcode budget" instead of just "budget." My comment was about making the flag description strings match up with this rule too, since right now they look like "Apply max extra budget for apps during simulation"

Reply:
good callout, see [flag description fix and e2e error line fix](https://github.com/algorand/go-algorand/pull/5354/commits/37ad9b02ab7adfe12e0f1214192cf9549c9ca037), thanks for point out!